src/Collection.php

```diff
@@ -520,9 +520,10 @@ public function find($filter = [], array $options = [])
             $options['typeMap'] = $this->typeMap;
         }
 
-        $operation = new Find($this->databaseName, $this->collectionName, $filter, $options);
+        //$operation = new Find($this->databaseName, $this->collectionName, $filter, $options);
+        //return $operation->execute($server);
 
-        return $operation->execute($server);
+        return new \MongoDB\Cursor($this, $filter, $options);
```

Comment:
> I just wanted to make some simple additions for ease of use, also compatible with legacy version.

Understandable. From the library's standpoint, we need to be very selective about what APIs we expose. Providing only a subset of the API is a slippery slope as additional BC layers might be requested down the line and the legacy driver has a number of poor design decisions that we absolutely do not want to encourage any longer. Having multiple APIs for the same functionality (e.g. [CRUD spec](https://github.com/mongodb/specifications/blob/master/source/crud/crud.rst) and legacy API) would also make the already-verbose [documentation](https://docs.mongodb.com/php-library/) harder to digest.

To that end, I'd rather promote [mongo-php-adapter](https://github.com/alcaeus/mongo-php-adapter) as a full compatibility layer.